src/patternfly/components/Toolbar/examples/Toolbar.md

```diff
@@ -34,6 +34,7 @@ Toolbar relies on groups (`.pf-c-toolbar__group`) and items (`.pf-c-toolbar__ite
 | `.pf-m-visible{-on-[breakpoint]}` | `.pf-c-toolbar > *` | Modifies toolbar element to be shown, at optional breakpoint. |
 | `.pf-m-align-right{-on-[breakpoint]}` | `.pf-c-toolbar > *` | Modifies toolbar element to align right, at optional breakpoint. |
 | `.pf-m-align-left{-on-[breakpoint]}` | `.pf-c-toolbar > *` | Modifies toolbar element to align left, at optional breakpoint. |
+| `.pf-m-sticky-top` | `.pf-c-toolbar-*` | Modifies toolbar element to be sticky to the top of its container. |
```

Comment:
Updated the class it applies to, though this table is for modifiers that apply to the toolbar items/children. Can you move this line down to the table under the "Overview" section? You can add this as the first modifier in that list (above the entry `.pf-m-expanded | .pf-c-toolbar__expandable-content |  Modifies expandable content section for the expanded state.`



```suggestion
| `.pf-m-sticky-top` | `.pf-c-toolbar` | Modifies toolbar element to be sticky to the top of its container. |
```

Reply:
Sure! Thanks for catching this.